src/reducers/rootReducer.js

```diff
@@ -1,5 +1,8 @@
 import { combineReducers } from "redux";
 import simpleReducer from "./simpleReducer";
+import userReducer from "../reducers/authReducers/socialAuthReducer";
 export default combineReducers({
```

Comment:
Always skip a line between the imports and components

Reply:
@LehruAngela  I have made this change